app/addons/replication/__tests__/helpers-tests.js

```diff
@@ -0,0 +1,34 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
```

Comment:
hmmm... i like the new folder `__test__` -- it is jest standard and it separates the old system clearly from the new